configs/config.json.cluster.example

```diff
@@ -208,17 +191,14 @@
       "cache_recent_forts": true
     },
     "pokemon_bag": {
-      "// if 'show_at_start' is true, it will log all the pokemons in the bag (not eggs) at bot start": {},
       "show_at_start": true,
-      "// if 'show_count' is true, it will show the amount of each pokemon (minimum 1)": {},
       "show_count": false,
-      "// if 'show_candies' is true, it will show the amount of candies for each pokemon": {},
       "show_candies": false,
-      "// 'pokemon_info' parameter define which info to show for each pokemon": {},
-      "// the available options are": {},
-      "// ['cp', 'iv_ads', 'iv_pct', 'ivcp', 'ncp', 'level', 'hp', 'moveset', 'dps']": {},
```

Comment:
Not sure how others feel, but I like in-example.config explanations. Most don't head over to our configuration_files.md and look at what is going on. It's purely a preference.